### PR TITLE
Issue234 add wea mea for testcases

### PR DESCRIPTION
**Pull request by dhblum**
For #234:

- Adds weather station to ``bestest_air`` test case

### CODE REVIEW

**Comment by dhblum** [comment]
This PR will also update ``bestest_hydronic`` and ``bestest_hydronic_heat_pump`` following https://github.com/open-ideas/IDEAS/issues/1188.

**Comment by dhblum** [comment]
All tests pass locally.

**Comment by dhblum** [comment]
All tests pass locally.  @JavierArroyoBastida Can you give this a reivew?

**Comment by JavierArroyoBastida** [comment]
Yes, thanks a lot for getting this implemented @dhblum! I'm working on the review today.

File: testcases/bestest_air/doc/index.html [hunk 1]
```diff
@@ -220,7 +220,7 @@ <h4>Rule-based or local-loop controllers (if included)</h4>
 </p>
 <p align=\"center\">
 <img alt=\"Control scheme diagram\"
-src="images/ControlSchematic_Ideal.png" width=600 />
+src=\"../../../doc/images/ControlSchematic_Ideal.png\" width=600 />
```
Comment (JavierArroyoBastida) [inline testcases/bestest_air/doc/index.html:223]: I can´t see the image in the html file because of this change

**Review by JavierArroyoBastida — CHANGES_REQUESTED** [review]
@dhblum all changes look good to me except for the small comment on the documentation of the bestest_air. Any idea why Travis tests are not triggered?

File: testcases/bestest_air/doc/index.html (hunk 1 above)
Comment (dhblum) [reply testcases/bestest_air/doc/index.html]: Thanks for catching this.  Didn't mean to make this change.  Fixed in https://github.com/ibpsa/project1-boptest/pull/255/commits/701e1e9e870c40652983f7317dc53ffebfd40168.

**Comment by dhblum** [comment]
Thanks @JavierArroyoBastida for the review.  I made the fix you pointed out.  

Travis changed their pricing model recently and we're sorting out how best to get resources to run tests.

**Comment by dhblum** [comment]
@JavierArroyoBastida If you don't mind, and if my fix works for you, can you approve the review?

**Review by JavierArroyoBastida — APPROVED** [review]
I can see the image now, thanks for fixing it @dhblum.

**Comment by dhblum** [comment]
Great, thanks!  I think we should merge https://github.com/ibpsa/project1-boptest/pull/263 first, then update this branch with needed changes and merge.

**Comment by JavierArroyoBastida** [comment]
Yep, I think that makes sense.

**Comment by dhblum** [comment]
Merged latest master, resolved conflicts, and passed all pre-commit checks after running ``pre-commit run --all-files``.  Can be merged after tests pass.

**Pull request merged**